perception/lidar_centerpoint/config/centerpoint.param.yaml

```diff
@@ -12,3 +12,4 @@
     iou_nms_target_class_names: ["CAR"]
     iou_nms_search_distance_2d: 10.0
     iou_nms_threshold: 0.1
+    yaw_norm_thresholds: [0.5, 0.0, 0.5]
```

Comment:
From the results of your analysis, I think `0.3` is enough to filter out incorrect orientation estimations. https://tier4.atlassian.net/l/cp/Nqcenf0b

Reply:
I will update the thresholds right away, but is it ok to put private links here?

Reply:
*The pointainting config was executable so I removed that permission with `chmod a-x`

Reply:
> I will update the thresholds right away, but is it ok to put private links here?

Yes. But please care about that the private name isn't included in the link.

Reply:
:bow:

Reply:
[1abf696](https://github.com/autowarefoundation/autoware.universe/pull/1962/commits/1abf69663346c6c934228fe22c73bcafb41538a1)